addons/l10n_din5008/static/src/scss/report_din5008.scss

```diff
@@ -58,6 +58,9 @@
                 }
             }
         }
+        .information_block tr:nth-child(2) td {
+            vertical-align: bottom;
+        }
```

Comment:
Wouldn't it simpler to just use a div ...

Reply:
yes you are right I just updated it.